### ISSUE TITLE
bbg irc feedback

### ISSUE THREAD

**Issue by foxnewsnetwork** [foxnewsnetwork/American-Lang220#20]
(3:01:13 PM) notola: anyone here interested in making visual novels? They're pretty easy to make compared to an actual game, but other game devs never seem interested in making them.
(3:01:29 PM) JakeCattrall left the room (quit: Quit: Leaving).
(3:02:53 PM) The account has disconnected and you are no longer in this chat. You will be automatically rejoined in the chat when the account reconnects.
(3:04:38 PM) The topic for #bbg is: Browser-based games channel for PBBG and HTML5 developers. If we don't respond right away, it's because we're busy coding. WWW: http://hashbbg.com/ Forums: http://bit.ly/da1eE5 Game List: http://bit.ly/eNxe2G
(3:04:43 PM) tsatse: ok thanks, I thought you were suggesting notola to use renpy
(3:04:53 PM) FuzzYspo0N: lol, wb notola
(3:05:12 PM) notola: lol sorry I missed everything, internet just disconnected -_-
(3:05:29 PM) Nibelung: http://i.imgur.com/Y37jv.png
(3:06:00 PM) FuzzYspo0N: no worries, i was just suggesting renpy makes it easy to make these visual novels, and the HEAVY content requirement means that : no skill , just story, and heavy content = no fun for most game devs i know
(3:06:15 PM) FuzzYspo0N: or are limited by lack of story and lack of large quantities of content/art
(3:06:26 PM) notola: I see
(3:06:33 PM) FuzzYspo0N: and to be honest, making them more than once would be super boring as a coder or something
(3:06:54 PM) notola: it's certainly not challenging once you have a framework setup
(3:07:01 PM) FuzzYspo0N: http://blog.scoutshonour.com/ this girls games, they are visual novel like but are games too
(3:07:03 PM) notola: even the art part since you can just recycle art
(3:07:16 PM) FuzzYspo0N: maybe, but who wants the same thing over and over ;)
(3:07:25 PM) FuzzYspo0N: her newest game : http://ahatestory.com/
(3:07:46 PM) FuzzYspo0N: and the counter part (older) http://scoutshonour.com/digital/
(3:08:34 PM) notola: see, this is what I've always wondered about; visual novels are all super light in terms of computation yet all of them seem to require installs and stuff
(3:08:44 PM) FuzzYspo0N: my point being : there are plenty game devs making these kinda things ;)
(3:08:47 PM) notola: wouldn't it make sense to have a builder / publication platform just on the cloud?
(3:08:55 PM) FuzzYspo0N: hmm, it might
(3:09:00 PM) FuzzYspo0N: I'm sure there are though
(3:09:14 PM) Cygnus_X [62c0528c@gateway/web/freenode/ip.98.192.82.140] entered the room.
(3:09:30 PM) QF-MichaelK: notola: Might as well make a full game
(3:09:42 PM) QF-MichaelK: notola: There are a lot of game devs that are short on contenet
(3:09:55 PM) tsatse: so how do you play in a visual novel ?
(3:10:04 PM) QF-MichaelK: content\* and there seem to be a lot of art people that are short on the tech side... may as well combine them
(3:10:27 PM) notola: tastse: you click and read through text while looking at pictures and occasionally make a decision
(3:10:55 PM) FuzzYspo0N: lol, spending on the kind of them of course
(3:10:56 PM) notola: QF-MichaelK : I know right, there should be some sort of github-like platform for visual novels
(3:11:12 PM) FuzzYspo0N: tsatse, that second one i linked ( digital, a love story ) is free, grab it and play
(3:11:29 PM) tsatse: thanks !
(3:11:46 PM) tsatse: oh yeah but it's not online...
(3:11:47 PM) FuzzYspo0N: i don't feel like this is a new avenue, there are hundreds of tools around for this (some online and most downloadable)
(3:12:23 PM) notola: alright I'm going to spend the next 30 minutes scaffolding out a quick sign-up site for a cloud-based visual novel github-like platform then see if there are actually people interested in this
(3:12:35 PM) FuzzYspo0N: sign up site? heh.
(3:12:56 PM) FuzzYspo0N: i hate those. prototype the engine first, then get people to try that - if the tools are good, the platform would do well
(3:12:58 PM) notola: yeah, I don't want to build stuff that no one ever uses anymore
(3:13:17 PM) FuzzYspo0N: or do you just mean a platform to push them onto? since there are thousands of those.
(3:13:41 PM) notola: both; like you'd build them on their too so you can "fork" other people's pictures and stuff
(3:13:54 PM) notola: _on there
(3:14:03 PM) FuzzYspo0N: good luck with that ;)
(3:14:12 PM) notola: do you think it's a workable idea, though?
(3:14:20 PM) FuzzYspo0N: i think its monolithic
(3:14:39 PM) FuzzYspo0N: and i think a sign up site is a good way to tell a big team somewhere else about the idea :P
(3:14:51 PM) FuzzYspo0N: make the tools first, if they are usable (properly) as a prototype
(3:14:57 PM) FuzzYspo0N: then a sign up site is a good idea
(3:15:00 PM) notola: lol if someone else does it, I'm fine with it
(3:15:08 PM) FuzzYspo0N: there is nothing worse than a cool startup that just vaporwares cos its huge
(3:15:17 PM) FuzzYspo0N: startup/idea/whatever
(3:15:25 PM) notola: besides, I've been mulling around with the idea / poking @ it for a while now, so I can probably get it out pretty fast
(3:15:27 PM) tsatse: not very handy to use a computer with only one hand
(3:15:33 PM) FuzzYspo0N: you basically just ruin your reputation :P
(3:16:27 PM) FuzzYspo0N: well do the prototype first, its way better than a crappy sign up for nothing ware :P
(3:16:34 PM) FuzzYspo0N: something tangible
(3:16:57 PM) notola: how "tangible"?
(3:17:39 PM) tsatse: it's not the spoon that bends notola
(3:18:21 PM) notola: like, I already have a couple simple jpgs that detail how I imagine the work-flow would be
(3:18:41 PM) FuzzYspo0N: notola, no, a usable tool. Something where I can make a visual story (rubbish one obviously)
(3:19:07 PM) FuzzYspo0N: as in, i want to know 'can i make my next novel with this'
(3:19:23 PM) FuzzYspo0N: if that can't be answered in a prototype, i won't ever come back to the service :P
(3:19:42 PM) Nibelung left the room (quit: Quit: Leaving.).
(3:19:45 PM) notola: you make a really good point there
(3:19:49 PM) *__|3b| suspects the old 'make games not engines' applies
(3:20:08 PM) FuzzYspo0N: agreed, but if you aren't already making games that applies a bit less :P
(3:20:16 PM) FuzzYspo0N: there are needs for tech too
(3:20:28 PM) |3b|: which you won't know if you don't make the games :p
(3:20:55 PM) FuzzYspo0N: and yea, second great point, if you aren't a maker or whatever your tool does, its really hard to make it great for users
(3:21:02 PM) |3b|: and which won't sell unless you have good demo games
(3:21:04 PM) FuzzYspo0N: like workflow and such is important
(3:21:36 PM) FuzzYspo0N: in other words , a game engine developer should spend more time making games with the tech than in QA, since the frustrations and requests come from there, not QA
(3:21:50 PM) FuzzYspo0N: (from a user point of view)
(3:22:07 PM) notola: that's true
(3:22:32 PM) FuzzYspo0N: so, you'd need someone who makes visual novels with all the other tools, testing
(3:22:35 PM) FuzzYspo0N: saying hey this is shitty
(3:22:45 PM) FuzzYspo0N: or ;they do x and it sucks, what about y - its better'
(3:22:55 PM) |3b|: dunno that you would need someone familiar with the other tools to start with
(3:23:13 PM) |3b|: harder to come up with new ideas that way :)
(3:23:14 PM) FuzzYspo0N: i think its important if you want to appeal to users of those tools
(3:23:30 PM) |3b|: just be prepared to hack in a lot of copied features later
(3:23:49 PM) notola: lol sounds fun
(3:23:57 PM) FuzzYspo0N: yea, well more that i meant its a good basis to start with: things users need/like, things they don't /wish for
(3:24:18 PM) FuzzYspo0N: how they work  can be up to you, but having someone who knows those details is what i meant
(3:24:25 PM) |3b|: people that already use your competition isn't the best target market to start with :)
(3:24:43 PM) FuzzYspo0N: i disagree, if you aren't competing whats the point?
(3:24:51 PM) FuzzYspo0N: or aren't competitive_
(3:25:04 PM) |3b|: i mean 'all the people who don't use the other tools' is probably a bigger market
(3:25:10 PM) FuzzYspo0N: I am suggesting to step above them by catering to users needs, rather than blindly making things with programmer sensibility
(3:25:26 PM) FuzzYspo0N: people that DONT make visual novels you mean :P i don't know
(3:25:44 PM) |3b|: so make something as good as you can without assuming it should work a particular way, before copying the competation
(3:25:56 PM) FuzzYspo0N: i never once said copy?
(3:25:59 PM) |3b|: right, the ones thatmight make them if the tools were better for example :)
(3:26:21 PM) |3b|: ok, 'before being influenced by' then :)
(3:27:17 PM) FuzzYspo0N: meh, i think tools are something that build on what works and scrap what doesn't
(3:27:27 PM) _**|3b| isn't saying kick someone off the team for knowing the other tools, just that it shouldn't be a priority to find someone
(3:28:06 PM) *__|3b| thinks it is too easy to get stuck in a local maximum and miss the big picture
(3:28:25 PM) FuzzYspo0N: sigh , way to miss the point
(3:28:28 PM) FuzzYspo0N: of what i meant
(3:28:30 PM) FuzzYspo0N: but yea
(3:28:58 PM) FuzzYspo0N: your idea is isolation from ANYHING out there and making a tool with 0 clue
(3:28:59 PM) FuzzYspo0N: good idea
(3:29:09 PM) FuzzYspo0N: (see, misinterpretation is weird)
(3:29:16 PM) |3b|: nope, sounds about right :)
(3:29:25 PM) FuzzYspo0N: pfft
(3:29:29 PM) FuzzYspo0N: good luck with that ;p
(3:30:01 PM) FuzzYspo0N left the room (quit: Quit: sleeptime).
(3:31:19 PM) notola: alright well, I just put up a crappy sign-up site for the github-for-visual-novel idea
(3:31:35 PM) notola: http://ffopenvn.heroku.com in case anyone is curious
(3:33:17 PM) |3b|: signing up for stuff is too hard :p
(3:33:30 PM) notola: lol
(3:34:18 PM) *_*|3b| is serious about that... though i guess a closed beta sort of justifies it
(3:34:29 PM) tsatse: with one hand, it's impossible
(3:34:56 PM) |3b|: put the signup after they write a few chapters and want to save... much more likely to get someone to sign up :)
(3:35:16 PM) notola: omg that's actually a pretty smart idea
(3:35:58 PM) |3b|: (but save it temporarily anyway, since losing work is a great way to make people hate you)

**Comment by foxnewsnetwork** [comment]
TL;DR : people are really interested in the idea of a visual-novel publication platform like github except for visual novels and think I should build this.
